Aberrus/Locales/ptBR.lua

```diff
@@ -1,106 +1,106 @@
 if not L then return end
 if L then
-	--L.dread_rift = "Rift" -- Singular Dread Rift
+	L.dread_rift = "Fenda" -- Singular Dread Rift
 end
 
 L = BigWigs:NewBossLocale("The Amalgamation Chamber", "ptBR")
 if L then
-	--L.custom_on_fade_out_bars = "Fade out stage 1 bars"
-	--L.custom_on_fade_out_bars_desc = "Fade out bars which belong to the boss that is out of range in stage 1."
-
-	--L.coalescing_void = "Run Away"
-	--L.shadow_convergence = "Orbs"
-	--L.molten_eruption = "Soaks"
-	--L.swirling_flame = "Tornadoes"
-	--L.gloom_conflagration = "Meteor + Run Away"
-	--L.blistering_twilight = "Bombs + Tornadoes"
-	--L.convergent_eruption = "Soaks + Orbs"
-	--L.shadowflame_burst = "Frontal Cone"
-
-	--L.shadow_and_flame = "Mythic Debuffs"
+	L.custom_on_fade_out_bars = "Esconder barras da primeira fase."
+	L.custom_on_fade_out_bars_desc = "Desaparece com as barras que pertencem ao boss que está fora de alcance na primeira fase."
+
+	L.coalescing_void = "Fuja"
+	L.shadow_convergence = "Orbes"
+	L.molten_eruption = "Soaks"
+	L.swirling_flame = "Tornados"
+	L.gloom_conflagration = "Meteoro + Fuja"
+	L.blistering_twilight = "Bombas + Tornados"
+	L.convergent_eruption = "Soaks + Orbes"
+	L.shadowflame_burst = "Cone Frontal"
+
+	L.shadow_and_flame = "Debuffs Míticos"
 end
 
 L = BigWigs:NewBossLocale("The Forgotten Experiments", "ptBR")
 if L then
-	--L.rending_charge_single = "First Charge"
-	--L.massive_slam = "Frontal Cone"
-	--L.unstable_essence_new = "New Bomb"
-	--L.custom_on_unstable_essence_high = "High Stacks Unstable Essence Say Messages"
-	--L.custom_on_unstable_essence_high_desc = "Say messages with the amount of stacks for your Unstable Essence debuff when they are high enough."
-	--L.volatile_spew = "Dodges"
-	--L.volatile_eruption = "Eruption"
-	--L.temporal_anomaly = "Heal Orb"
-	--L.temporal_anomaly_knocked = "Heal Orb Knocked"
+	L.rending_charge_single = "Primeira Investida"
+	L.massive_slam = "Cone Frontal"
+	L.unstable_essence_new = "Nova Bomba"
+	L.custom_on_unstable_essence_high = "Mensagens de Acumulos Altos de Essência Instável"
+	L.custom_on_unstable_essence_high_desc = "Diga mensagens com a quantidade de acúmulos do seu debuff de Essência Instável quando eles estiverem altos o suficiente."
+	L.volatile_spew = "Esquivar"
+	L.volatile_eruption = "Erupção"
+	L.temporal_anomaly = "Orbe de Cura"
+	L.temporal_anomaly_knocked = "Orbe de Cura Empurrado"
 end
 
 L = BigWigs:NewBossLocale("Assault of the Zaqali", "ptBR")
 if L then
 	-- These are in-game emotes and need to match the text shown in-game
 	-- You should also replace the comment (--) with the full emote as it shows in-game
-	--L.zaqali_aide_north_emote_trigger = "northern battlement" -- Commanders ascend the northern battlement!
-	--L.zaqali_aide_south_emote_trigger = "southern battlement" -- Commanders ascend the southern battlement!
+	L.zaqali_aide_north_emote_trigger = "Reforço Norte" -- Commanders ascend the northern battlement!
```

Comment:
We would like the comments here (--) replaced with the full ptBR text copy paste from the in-game emote.